apps/skilavottord/ws/src/app/app.module.ts

```diff
@@ -38,6 +39,10 @@ const autoSchemaFile = environment.production
       useClass: SequelizeConfigService,
     }),
     AuthModule,
+    AuthJwtModule.register({
+      audience: null,
+      issuer: 'https://identity-server.dev01.devland.is',
```

Comment:
This config needs to be environment based.

Reply:
fixed